### DISCUSSION TITLE
这个sofa-rpc和dubbo对比有什么区别和优势？

### DISCUSSION THREAD

**Discussion by lwb314** [sofastack/sofa-rpc#1317] [General]
性能和dubbo有做过对比吗？之前微服务框架一直用的是dubbo，从阿里开源的版本切换到apache版本。有什么优势能让开发人员选择更换技术架构吗？

**Comment by OrezzerO** [comment]
可以参考 #34 .  Dubbo 是一款优秀的RPC 框架, 如果已将在使用 Dubbo ,并且没有什么痛点的话, 我认为是没必要更换的.

技术选型需要考虑点比较多, 技术生态是比较重要的一个点.  可以整体考虑使用蚂蚁的生态,还是用阿里巴巴的生态 ,光 RPC 一个点, SOFA-RPC 和 Dubbo 并不会有太大出入.

另外 SOFA-RPC 和新版本 Dubbo 可以通过 Triple 协议互通, 同时存在异构的框架也是可以的.
